### PR TITLE
[Databricks] Remove some custom options

### PR DESCRIPTION
**Pull request by fengzhou-msft**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
add to S168

File: src/databricks/azext_databricks/_params.py [hunk 1]
```diff
@@ -23,17 +22,9 @@ def load_arguments(self, _):
...
-        c.argument('enable_no_public_ip', arg_type=get_three_state_flag(), help='Set true to disallow public IP. Default is false')
-        c.argument('load_balancer_backend_pool_name', help='The name of load balancer backend pool name to create.')
-        c.argument('load_balancer_id', options_list=['--load-balancer'], help='Load balancer name or resource ID.')
-        c.argument('relay_namespace_name', help='The name of relay namespace. Default value is "dbrelay" as prefix, followed by random digits and letters.')
-        c.argument('storage_account_name', help='Storage account name to create. Default value is "dbstorage" as prefix, followed by random digits and letters.')
-        c.argument('storage_account_sku_name', options_list=['--storage-account-sku'], arg_type=get_enum_type(['Standard_LRS', 'Standard_ZRS', 'Standard_GRS', 'Standard_RAGRS', 'Premium_LRS']), default='Standard_GRS', help='Storage account SKU.')
-        c.argument('vnet_address_prefix', default='10.139', help='The vnet address prefix.')
+        c.argument('custom_virtual_network_id', options_list=['--vnet'], arg_group='Custom VNET', help='Virtual network name or resource ID.')
```
Comment (yungezz) [inline src/databricks/azext_databricks/_params.py:25]: do you want to comment them out for future each pick up?

Comment (fengzhou-msft) [reply src/databricks/azext_databricks/_params.py:25]: We can rely on this PR to put them back, keep it clean for this release.

**Comment by fengzhou-msft** [comment]
> Is there any special reason for remove/rename these parameters ? And it seems to be a BREAKING CHANGE pr.

Parameters removal is requested by service team as they are still in private preview. Renaming is to align the names with Powershell and also make options shorter.

**Pull request merged**